src/components/wallet/views/loggedInContent.tsx

```diff
@@ -29,14 +29,13 @@ export default function loggedInContent(this: Wallet) {
           : 'Use Testnet'}
       </button>
     </div>,
-
```

Comment:
lets keep this whitespace or remove the other newlines in the array to keep it consistent